### PR TITLE
[ticket/14339] Properly state PHP version support

### PR DESCRIPTION
**Pull request by marc1706**
Ticket: https://tracker.phpbb.com/browse/PHPBB3-14339

Master version of #4063

### CODE REVIEW

**Comment by Nicofuma** [comment]
you have to rebase

**Comment by marc1706** [comment]
Fair enough

File: phpBB/docs/INSTALL.html [hunk 1]
```diff
@@ -147,7 +147,7 @@
 			<li>Oracle</li>
 		</ul>
 		</li>
-		<li><strong>PHP 5.4.0+</strong> with support for the database you intend to use.</li>
+		<li><strong>PHP 5.4.0+</strong> and <strong>PHP < 7.0</strong> with support for the database you intend to use.</li>
```
Comment (Crizz0) [inline phpBB/docs/INSTALL.html]: Not 7.1 like in composer.json

Comment (marc1706) [inline phpBB/docs/INSTALL.html]: Sorry, failed at that

Comment (Crizz0) [inline phpBB/docs/INSTALL.html]: No excuse needed. We've found that early enough. :)

**Comment by marc1706** [comment]
@Nicofuma should be ready to go now

**Comment by Nicofuma** [comment]
Ok thanks. I'll take care of that tonight

**Pull request merged**